Data-Structure/Collection-In-Java/CollectionOfEmployee/Company.java

```diff
@@ -0,0 +1,39 @@
+package CollectionOfEmployee;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Niraj Gupta
+ * Class to represent the company which have employee.
+ *
+ */
+public class Company{
+	private Map<String, Employee> company = new HashMap<String, Employee>();

```

Comment:
in my view a company instance should be a collection of company name/id and list of employees in it